ads/google/doubleclick.js

```diff
@@ -55,18 +44,15 @@ export function doubleclick(global, data) {
 
   centerAd(global);
 
-  const gptFilename = selectGptExperiment(data);
-
-  writeAdScript(global, data, gptFilename);
+  writeAdScript(global, data);
 }
 
 /**
  * @param {!Window} global
  * @param {!Object} data
- * @param {!GladeExperiment} gladeExperiment
- * @param {!string} url
  */
-function doubleClickWithGpt(global, data, gladeExperiment, url) {
+function doubleClickWithGpt(global, data) {
+  const url = 'https://www.googletagservices.com/tag/js/gpt.js';
```

Comment:
Nit: Moving this definition closer to the usage or inlining avoids someone having to search around for this variable.

Reply:
just removed the declaration and inlined it below